Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -254,15 +253,22 @@ final class ChangeStreamSpecTests: MongoSwiftTestCase {
             }
             print("\n------------\nExecuting tests from file \(testName)...\n")
             for var test in testFile.tests {
-                let testTopologies = test.topology.map { TopologyDescription.TopologyType(from: $0) }
-                guard testTopologies.contains(topology) else {
-                    print(unsupportedTopologyMessage(testName: test.description, topology: topology))
-                    continue
-                }
+                let testRequirements = TestRequirement(
+                    minServerVersion: test.minServerVersion,
+                    acceptableTopologies: test.topology.map { TestTopologyConfiguration(from: $0) }
+                )
 
-                guard version >= test.minServerVersion else {
-                    print("Skipping test case \"\(test.description)\": minimum required server " +
-                        "version \(test.minServerVersion) not met.")
+                let unmetRequirements = try globalClient.meetsRequirements(testRequirements)
+                guard unmetRequirements == nil else {
+                    switch unmetRequirements {
+                    case .minServerVersion:
+                        print("Skipping test case \"\(test.description)\": minimum required server " +
```

Comment:
It would be fine just to replace this message with the one from `printRequirementNotMetMessage`.

Reply:
done

Reply:
done